src/host/VtIo.cpp

```diff
@@ -397,3 +397,37 @@ void VtIo::_ShutdownIfNeeded()
         ServiceLocator::RundownAndExit(ERROR_BROKEN_PIPE);
     }
 }
+
+// Method Description:
+// - Tell the vt renderer to begin a resize operation. During a resize
+//   operation, the vt renderer should _not_ request to be repainted during a
+//   text buffer circling event. Any callers of this method should make sure to
+//   call EndResize to make sure the renderer returns to normal behavior.
+//   See GH#1795 for context on this method.
+// Arguments:
+// - <none>
+// Return Value:
+// - <none>
+void VtIo::BeginResize()

```

Comment:
Uh, maybe this method should just return an RAII object that stops the resize when it is destroyed like what we envision good locks to do.

Reply:
You know, I almost wrote it that way. I'll fix it.

Reply:
Moved to #2162